synapse/lib/storm.py

```diff
@@ -2732,28 +2732,43 @@ def help(self, mesg=None):
         self.exited = True
         return False
 
+    def _wrap_txt(self, txt, wth):
```

Comment:
This function is a little hard to follow with the abbreviated variable names but i think you're replicating textwrap.wrap here?

Reply:
That's right, I was trying to avoid importing `textwrap` to maximum optimization for its specific task. Also wasn't sure how verbose or concise variable names need/should be so went with this.

Reply:
Unless we're doing something in a really hot loop (which this isn't), it's generally acceptable to use the stdlib.

My observation is that variable names are often abbreviated but still descriptive. A lot of variable names are abbreviated to 4 characters as long as they're still descriptive ('iden', 'nexs', etc). In this case, I wasn't quite sure what 'ls' or 'cl' were supposed to be. I figured out 'ls' is lines but never understood 'cl'.

Reply:
current line

Reply:
DM'd about var names being just a hair more verbose ( def like the terse syntax ). I'd prefer to use this split based implementation rather than dragging in the `re` module which the `textwrap` module does. Some light-weight / self contained text formatting seems fine :+1: